doc/tutorials/isingmodel.ipynb

```diff
@@ -0,0 +1,612 @@
+{
```

Comment:
<p>Maybe: "The minimun energy should be -2 for the [1,1,-1] spin configuration, which corresponds to angles of [0,0,0] and [0,pi,0] for the spin 2 and spin 3 respectively."</p>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>